test/bdd/cmd/issuer/main.go

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
tip: I think you could have used `httptest.Server` instead

Reply:
Linux doen't support DNS resolution to host machine. I was facing issue with calling host port from inside docker container in github actions - https://github.com/docker/for-linux/issues/264